helm/hpcc/templates/NOTES.txt

```diff
@@ -1,10 +1,8 @@
 {{- $defaultImage :=  include "hpcc.imageName" (dict "root" $ "me" dict ) | quote }}
 {{- $defaultVersion :=  .Values.global.image.version | default .Chart.Version }}
 Thank you for installing the HPCC chart version {{ .Chart.Version }} using image {{ $defaultImage }}
+{{ include "hpcc.printWarnings" (dict "root" .) | indent 4 -}}
```

Comment:
Can lines 6-20 also be deleted?  I think they are only used to update $match which was used for reporting warnings.

Reply:
You're right. I'll remove these lines.